static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -497,6 +502,13 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
                         ? rowWithAudience.audienceOffset
                         : 0;
 
+                    const rowWithUserCohort = rows.find(
```

Comment:
ah, so this is a "the first row that has a value affects the whole test" type parameter?

Reply:
indeed, it's far from ideal and we can do something better later